pytorch_lightning/plugins/training_type/sharded.py

```diff
@@ -28,10 +31,23 @@
 
 class DDPShardedPlugin(DDPPlugin):
 
+    def __init__(
+        self,
+        parallel_devices: Optional[List[torch.device]] = None,
+        num_nodes: int = 1,
+        cluster_environment: ClusterEnvironment = None,
+        sync_batchnorm: bool = False,
+        is_fp16: bool = False,
```

Comment:
We should be able to look this up from the trainer, so it doesn't need to be passed again here

Reply:
> We should be able to look this up from the trainer, so it doesn't need to be passed again here

you are absolutely right!